### PR TITLE
Test variation order change

### PR DESCRIPTION
**Pull request by aarsilv**
Temporary branch to confirm changing the order of variations does not affect tests

### CODE REVIEW

File: ufc/flags-v1.json [hunk 1]
```diff
@@ -505,6 +505,101 @@
       ],
       "totalShards": 10000
     },
+    "kill-switch-2": {
```
Comment (aarsilv) [inline ufc/flags-v1.json:508]: copy of `"kill-switch"` above, but with the variation order flipped

File: ufc/tests/test-case-kill-switch-flag-2.json [hunk 2]
```diff
@@ -0,0 +1,949 @@
+{
+  "flag": "kill-switch-2",
```
Comment (aarsilv) [inline ufc/tests/test-case-kill-switch-flag-2.json:2]: copy of `test-case-kill-switch-flag.json` but with the flag name changed to use the one with flipped variation order

File: ufc/flags-v1.json [hunk 3]
```diff
@@ -505,6 +505,101 @@
...
+      "key": "kill-switch",
```
Comment (typotter) [inline ufc/flags-v1.json:509]: Incorrect key.

Comment (aarsilv) [reply ufc/flags-v1.json:509]: 🤦‍♂️

**Pull request closed without merging**